<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Rock.png</pixmap>
   </property>
  </widget>
  <widget class="QSlider" name="rangeScale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="numSections">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="value">
    <number>51</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="value">
    <number>51</number>
   </property>
  </widget>
  <widget class="QPushButton" name="highBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>650</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cook High Res</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>690</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Voxel Res:</string>
   </property>
  </widget>
  <widget class="QSlider" name="minRot">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>580</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="maxRot">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>620</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>-9</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Rotation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Rotation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sections</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="voxelVal">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>690</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>0.005000000000000</double>
   </property>
   <property name="value">
    <double>0.003000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rangeScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rangeScale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numSections</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numSections</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>minRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>591</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>maxRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numSections</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
